app/helpers/standard_form_builder.rb

```diff
@@ -287,7 +287,8 @@ def has_many_field(attr, html_options = {}) # rubocop:disable Naming/PredicateNa
     ].compact.join(" ")
     html_options[:class] += " is-invalid" if errors_on?(attr)
 
-    add_css_class(html_options, "multiselect")
+    add_css_class(html_options, "multiselect tom-select")
+    html_options[:data] = {chosen_no_results: I18n.t("global.chosen_no_results")}
```

Comment:
For multi selects we have TomSelect, a tom select is initialized on every element with the `tom-select` class. For some reason the StandardFormBuilder did not add this class for `has_many_fields`

Reply:
Doesn't this change a lot of things in all hitobito instances if we add this to every affected form?

Reply:
Yes it does... kinda, as far as last time I upgraded bootstrap from 2 to 5 I didn't find a lot of multiselect fields in all wagons or core Features anyways.

Additionally this only changes the multiselect fields to look good, and does not really change any of the behaviour. All other options tom-select offers like search inside the multi select fields and so on, are not initialized by default in out implementation.

With TomSelect:
![image](https://github.com/user-attachments/assets/a10716a9-fca6-4c65-a8b6-c01c89e9036d)

Without TomSelect:
![image](https://github.com/user-attachments/assets/2f9097b8-4de6-4219-9f64-3058e0a8ae1a)